dev-python/b2sdk/b2sdk-1.2.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+inherit distutils-r1
+
+DESCRIPTION="The client library for BackBlaze's B2 product"
+HOMEPAGE="https://github.com/Backblaze/b2-sdk-python"
+SRC_URI="https://github.com/Backblaze/b2-sdk-python/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+PATCHES=(
+	"${FILESDIR}/${P}_skip-integration-test.patch"
+	"${FILESDIR}/${P}_skip-license-header-test.patch"
+)
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		>=dev-python/arrow-0.8.0[${PYTHON_USEDEP}]
+		>=dev-python/logfury-0.1.2[${PYTHON_USEDEP}]
+		>=dev-python/requests-2.9.1[${PYTHON_USEDEP}]
+		>=dev-python/tqdm-4.5.0[${PYTHON_USEDEP}]
+	')
+"
+
+distutils_enable_tests pytest
+
+BDEPEND+=" test? (
+	$(python_gen_cond_dep '
+		>=dev-python/pytest-6.1.1[${PYTHON_USEDEP}]
```

Comment:
d_e_t does pytest for you here at least

Reply:
I think I had some issues but likely due to putting the d_e_t call in the wrong place. I'll fix this.